pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -202,6 +202,10 @@ class PauliX(Observable, Operation):
 
     _queue_category = "_ops"
 
+    def __init__(self, *params, wires=None, do_queue=True, id=None):
```

Comment:
I don't think the docs will break, since we render the doc-string from the class. We could wrap this up into `(*args, **kwargs)`. I know Pylint complains when you have 6 or more arguments (I think 🤔). Maybe its okay to keep this, and if we end up with more, we swap to that then?

Reply:
I just don't like that there are 3 new constructors to only add this one lil thing. more code to keep track of, more mess in general. if we merge this today, the odds are so low that we ever come back to swap anything. I was tempted to say wouldn't it be great if these all shared some common, inherited code (aka a common Pauli module!) but realized that's also just adding more mess. once again, we find ourselves choosing the lesser of two evils. I'm fine to leave this as is.